local-running/build.gradle

```diff
@@ -60,6 +60,10 @@ application {
 	]
 }
 
+tasks.named('run', JavaExec) {
+	environment 'AWS_EMF_ENVIRONMENT', 'Local'
```

Comment:
Should this happen in the same place as `AWS_XRAY_CONTEXT_MISSING` above?

(Not sure which place is better, but it seems weird that they aren't the same?)

Reply:
Oh, I missed that. No idea which is better. I'll unify them though.